Documentation/project-docs/dogfooding.md

```diff
@@ -117,7 +117,8 @@ make it self-contained
   <PropertyGroup>
```

Comment:
Could you please re-number the steps in "2: Self-contained" as well while you are on it?

Reply:
sure, I missed that